main.cpp

```diff
@@ -0,0 +1,228 @@
+// Configuration editor
+// Command line utiltiy to create configuration files
+// Valid arguments are init and edit
+
+#include <iostream>
+#include <string>
+#include <fstream>
+#include <stdio.h>
+
+
+using std::cout;
+using std::endl;
+using std::string;
+using std::cin;
+
+const string CONFIG_FILE_PATH = "/users/bconway/SourceTree/School/CPT-180-27-Assignment-2-Config-Generator/app.cfg";
+
+
+int main(int argc, char *argv[]) {
+
+    string argument1 = "";
+    // Make sure that the argument isn't empty
+    if (argv[1] != nullptr) {
```

Comment:
Glad that I could help. I struggled with that for a while because I wanted to prompt the user for each setting of they used edit without a second argument.



Sent from my Verizon Wireless 4G LTE smartphone


-------- Original message --------
From: Samantha Shurie <notifications@github.com>
Date: 2/9/2016 6:23 PM (GMT-05:00)
To: WCCCEDU/CPT-180-27-Assignment-2-Config-Generator <CPT-180-27-Assignment-2-Config-Generator@noreply.github.com>
Cc: Brian Conway <brian@conwayfamily.us>
Subject: Re: [CPT-180-27-Assignment-2-Config-Generator] First Commit (#2)


In main.cpp<https://github.com/WCCCEDU/CPT-180-27-Assignment-2-Config-Generator/pull/2#discussion_r52391846>:

> +#include <stdio.h>
> +
> +
> +using std::cout;
> +using std::endl;
> +using std::string;
> +using std::cin;
> +
> +const string CONFIG_FILE_PATH = "/users/bconway/SourceTree/School/CPT-180-27-Assignment-2-Config-Generator/app.cfg";
> +
> +
> +int main(int argc, char *argv[]) {
> +
> +    string argument1 = "";
> +    // Make sure that the argument isn't empty
> +    if (argv[1] != nullptr) {


I was having an issue getting my program to run without an argv[2]. Using this same kind of if statement with argv[2] helped fix that, so I have to say, thank you!! Following through your code helped me fix that.

—
Reply to this email directly or view it on GitHub<https://github.com/WCCCEDU/CPT-180-27-Assignment-2-Config-Generator/pull/2/files#r52391846>.